content/dstu2/diagnostic-report.md

```diff
@@ -19,8 +23,14 @@ Search for DiagnosticReports that meet supplied query parameters:
 ------------------|-----------------|---------------------------------------------------------------|---------------------------------------------------------
 `patient`         | This or subject |[`reference`](http://hl7.org/fhir/dstu2/search.html#reference) | The subject of the report if a patient. Example: `12345`
 `subject:Patient` | This or patient |[`reference`](http://hl7.org/fhir/dstu2/search.html#reference) | The subject (Patient) of the report. Example: `12345`
+`date`            | N               | [`date`]      | Date range into which the diagnostic report falls (effectiveDateTime). Must be present once and prefixed by 'ge' or present twice and prefixed by 'ge'/'lt'. EG: `date=ge2014-09-24T12:00:00.000Z` `&date=lt2015-10-24T12:00:00.000Z`
```

Comment:
'ge'/'lt' is currently rendering it to ‘ge’/’lt’. The third quotation is showing up as closing quote rather than opening quote. If you pad "/" with a space either side it would show up correct. I tested this locally and it shows up correct.